### ISSUE TITLE
Chat text is drawn above NPC Dialogue overlay background

### ISSUE THREAD

**Issue by obj-obj** [Wynntils/Wynntils#1187]
If the minecraft UI library has z-indexes, maybe it could be fixed by setting the z-index higher?

Example:
![2023-02-20_20 00 16](https://user-images.githubusercontent.com/57120300/220244388-1d0f21f5-d847-4d88-8c88-bcd128b03950.png)

**Comment by magicus** [comment]
I think the basic problem here is that the default setting of the NPC dialogue overlay box overlaps with the chat.

**Comment by obj-obj** [comment]
> I think the basic problem here is that the default setting of the NPC dialogue overlay box overlaps with the chat.

Yeah, but just drawing the dialogue box on top of the chat would fix that. I kinda like the position it's in rn, just not that I can't read anything if there are too many messages shown

**Comment by magicus** [comment]
Do you want the NPC dialogue box to mask out everything that is behind it?

**Comment by obj-obj** [comment]
> Do you want the NPC dialogue box to mask out everything that is behind it?

Yeah, I might just make a PR for it if im bored

**Comment by magicus** [comment]
That's just a setting, the background opacity. Set it to 100% and you get what you want.

**Comment by obj-obj** [comment]
> That's just a setting, the background opacity. Set it to 100% and you get what you want.

The problem is that even if you set the opacity to 100%, the chat text still renders over the background, preventing it from being blocked out. The opacity is at 80% in this screenshot but the chat text clearly isn't affected by that

**Comment by magicus** [comment]
Ah, I see.
